src/DGtal/io/Display2DFactory.ih

```diff
@@ -1453,6 +1453,39 @@ void DGtal::Display2DFactory::draw( DGtal::Board2D & board,
 }
 // Object
 
+// CubicalComplex
+template < typename TKSpace, typename TCellContainer >
+inline 
+void 
+DGtal::Display2DFactory::draw( DGtal::Board2D & board, 
+             const DGtal::CubicalComplex<TKSpace, TCellContainer> & cc )
+{
+  FATAL_ERROR( cc.dimension == 2);
+  typedef DGtal::CubicalComplex<TKSpace, TCellContainer> CC;
+  typedef typename CC::KSpace               KSpace;
+  typedef typename CC::Point                Point;
+  typedef typename CC::Cell                 Cell;
+  typedef typename CC::CellMapConstIterator CellMapConstIterator;
+  typedef typename CC::ConstIterator        ConstIterator;
+  typedef typename KSpace::Integer          Integer;
+  for ( ConstIterator it = cc.begin(), itE = cc.end(); it != itE; ++it )
+    { // We are not using operator << for cells in order to use the
+      // style specified for the cubical complex.
+      Point kx = cc.space().uKCoords( *it );
+      float x = (float) ( NumberTraits<Integer>::castToInt64_t( kx[0] ) >> 1 );
+      float y = (float) ( NumberTraits<Integer>::castToInt64_t( kx[1] ) >> 1 );
```

Comment:
I prefer not to do it in this PR, but to do it properly in a separate PR, along with 3D view. I have the feeling that I may run into trouble (because of shifts problem). So perhaps we can raise an issue here.